app/javascript/components/gtl/DataTable.jsx

```diff
@@ -3,6 +3,7 @@ import PropTypes from 'prop-types';
 
 import { renderDataTableToolbar } from './utils';
 import { renderPagination } from './utils';
+import './styles.scss';
```

Comment:
I'm not sure if we want to import styling into react :confused: @himdel?

Reply:
I guess this is related to the CSS changes above, so a separate PR please. But in general we are not importing CSS into jsx files...

Reply:
I think css imports are fine for component css, as long as that css is independent of the rest.

Reply:
AFAIK we don't do that, so I don't think we should introduce it now